### DISCUSSION TITLE
CONNECTION FAILURE: connect ECOONNREFUSED

### DISCUSSION THREAD

**Discussion by Genne23v** [nocodb/nocodb#4150] [Q&A]
I'm trying to run cypress integration test. But the test can't move on due to SQL connection error. 
I'm running MySql with docker, and nocodb backend and frontend. 
What am I missing here? 

![Screen Shot 2022-10-21 at 3 08 55 PM](https://user-images.githubusercontent.com/59734889/197273449-5a15a372-5e7f-4073-a786-80fe0ecf335a.png)

**Comment by bcakmakoglu** [comment]
If your db is running on docker, use `host.docker.internal` instead of `localhost` as host address

**Reply by Genne23v** [reply]
I saw that post and tried with host.docker.internal. But it says `ENOTFOUND`. This is my docker running. 

<img width="1008" alt="Screen Shot 2022-10-21 at 3 33 47 PM" src="https://user-images.githubusercontent.com/59734889/197275505-b4d62092-8e02-410a-9426-c1c5ee660550.png">

**Reply by twicros** [reply]
same problem. Do you fix it?
